test/cluster.go

```diff
@@ -33,16 +33,30 @@ func NewGetMemberClusters(memberClusters ...*cluster.CachedToolchainCluster) clu
 	}
 }
 
+type Modifier func(toolchainCluster *cluster.CachedToolchainCluster)
+
+func WithClusterRoleLabel(labelKey string) Modifier {
+	return func(cluster *cluster.CachedToolchainCluster) {
+		cluster.Config.Labels[labelKey] = "" // we don't care about the label value, only the key is used
+	}
+}
+
 func NewMemberCluster(t *testing.T, name string, status corev1.ConditionStatus) *cluster.CachedToolchainCluster {
```

Comment:
I would rename it to 
```suggestion
func NewMemberClusterWithTenantRole(t *testing.T, name string, status corev1.ConditionStatus) *cluster.CachedToolchainCluster {
```
to make it explicit

Reply:
makes sense, change in https://github.com/codeready-toolchain/host-operator/pull/736/commits/6b01926137a7f5c783ed0b886e61eaba8cc13642